ibm/service/vpc/resource_ibm_is_virtual_endpoint_gateway.go

```diff
@@ -307,6 +307,10 @@ func resourceIBMisVirtualEndpointGatewayCreate(d *schema.ResourceData, meta inte
 	}
 
 	d.SetId(*result.ID)
+	_, err = isWaitForVirtualEndpointGatewayAvailable(sess, d.Id(), d.Timeout(schema.TimeoutUpdate))
```

Comment:
should we use schema.TimeoutCreate?

Reply:
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/4206/files#diff-a0d9b3493bb9618e40ba9b89e9cd9f93b54b8ba4de246bf6f6701cbfbbf72f53R62
Timeouts: &schema.ResourceTimeout is already defined

Reply:
why we have used update we shd use create right

Reply:
updated